src/components/DropdownBox.js

```diff
@@ -0,0 +1,49 @@
+import Interactive from 'react-interactive'
+import { h, div } from 'react-hyperscript-helpers'
+import { icon } from 'src/components/icons'
+import * as Style from 'src/libs/style'
+
+
+const styles = {
```

Comment:
I really like this pattern. I wouldn't mind taking some time to convert everything to do this.